### PR TITLE
🌐 Add Portuguese translation for `docs/pt/docs/advanced/fastapi-cli.md`

### PR DESCRIPTION
**Pull request by ayr-ton**
Translation of `docs/en/docs/fastapi-cli.md`

### CODE REVIEW

**Comment by tiangolo** [comment]
📝 Docs preview for commit 994c6829d20a1067a36143147cd91162a168d1c7 at: https://1d8c2f53.fastapitiangolo.pages.dev

**Review by nazarepiedady — APPROVED** [review]

File: docs/pt/docs/fastapi-cli.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
+# FastAPI CLI
+
+**FastAPI CLI** é um programa `fastapi` por linha de comando que você pode usar para rodar sua app FastAPI, gerenciar seu projeto FastAPI e mais.
+
+Quando você instala o FastAPI (ex.: com `pip install fastapi`), isso inclui um pacote chamado `fastapi-cli`. Esse pacote disponibiliza o comando `fastapi` no terminal.
+
+Para rodar sua app FastAPI em desenvolvimento, você pode usar o comando `fastapi dev`:
```
Comment (SamuelBFavarin) [inline docs/pt/docs/fastapi-cli.md:7]: As a brazilian portuguese native speaker, this phrase fits better with this version:

Para rodar sua app FastAPI em desenvolvimento, você pode usar o comando `fastapi dev`:
```suggestion
Para rodar seu app FastAPI em desenvolvimento, você pode usar o comando `fastapi dev`:
```

File: docs/pt/docs/fastapi-cli.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+
+## `fastapi run`
+
+Quando você rodar `fastapi run`, isso executará em modo de produção por padrão.
+
+Este modo terá **recarregamento automático desativado** por padrão.
+
+Isso irá escutar no endereço de IP `0.0.0.0`, o que significa todos os endereços IP disponíveis, dessa forma o programa estará acessível publicamente para qualquer um que consiga se comunicar com a máquina. Isso é como você normalmente roda em produção em um contêiner, por exemplo.
+
+Em muitos casos você pode ter (e deveria ter) um "proxy de saída" tratando HTTPS no topo, isso dependerá em como você fará o deploy da sua aplicação, seu provedor pode fazer isso pra você ou talvez seja necessário fazer você mesmo.
```
Comment (SamuelBFavarin) [inline docs/pt/docs/fastapi-cli.md:81]: I think this version sounds better:
```suggestion
Em muitos casos você pode ter (e deveria ter) um "proxy de saída" tratando HTTPS no topo, isso dependerá de como você fará o deploy da sua aplicação, seu provedor pode fazer isso pra você ou talvez seja necessário fazer você mesmo.
```

File: docs/pt/docs/fastapi-cli.md [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+<font color="#4E9A06">INFO</font>:     Started server process [<font color="#06989A">2265873</font>]
+<font color="#4E9A06">INFO</font>:     Waiting for application startup.
+<font color="#4E9A06">INFO</font>:     Application startup complete.
+```
+
+</div>
+
+Aquele commando por linha de programa chamado `fastapi` é o **FastAPI CLI**.
+
+FastAPI CLI pega o caminho para seu programa Python, detecta automaticamente a variável que contém o FastAPI (comumente nomeada `app`), como importá-la e então servi-la.
```
Comment (SamuelBFavarin) [inline docs/pt/docs/fastapi-cli.md:59]: I think we could include the article "O" at the beginning of this phrase:
```suggestion
O FastAPI CLI recebe o caminho do seu programa Python, detecta automaticamente a variável com o FastAPI (comumente nomeada `app`) e como importá-la, e então a serve.
```

File: docs/pt/docs/fastapi-cli.md [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+
+Para produção você usaria `fastapi run` no lugar. 🚀
+
+Internamente, **FastAPI CLI** usa <a href="https://www.uvicorn.org" class="external-link" target="_blank">Uvicorn</a>, um servidor ASGI de alta performance e pronto para produção. 😎
+
+## `fastapi dev`
+
+Quando você roda `fastapi dev`, isso vai executar em modo de desenvolvimento.
+
+Por padrão, teremos **recarregamento automático** ativo, então o programa irá recarregar o servidor automaticamente quando você fizer mudanças no seu código. Isso usa muitos recursos e pode ser menos estável que sem isso, você deve usar esse modo somente em desenvolvimento.
```
Comment (SamuelBFavarin) [inline docs/pt/docs/fastapi-cli.md:69]: What do you think about this option?

```suggestion
Por padrão, teremos o **recarregamento automático** ativo, então o programa irá recarregar o servidor automaticamente toda vez que você fizer mudanças no seu código. Isso usa muitos recursos e pode ser menos estável. Você deve apenas usá-lo em modo de desenvolvimento.
```

**Review by SamuelBFavarin — COMMENTED** [review]
Nice job, I just left few suggestions.

**Comment by tiangolo** [comment]
📝 Docs preview for commit 79b575993260088244e9f76990ac7c5728f889a8 at: https://410082ac.fastapitiangolo.pages.dev

**Comment by tiangolo** [comment]
📝 Docs preview for commit 60fa3a4bcdf87f12d73b050b2312702ac0970548 at: https://80cd29d2.fastapitiangolo.pages.dev

**Comment by nazarepiedady** [comment]
@tiangolo, @SamuelBFavarin, @alejsdev, do not you think the pull request is ready for merge?

**Review by SamuelBFavarin — APPROVED** [review]

**Comment by alejsdev** [comment]
Great! thanks for the contribution :partying_face: @ayr-ton 
And thanks to everyone for the review :v:

**Pull request merged**